tools/src/main/java/org/apache/kafka/tools/ClientMetricsCommand.java

```diff
@@ -238,24 +238,25 @@ public ClientMetricsCommandOptions(String[] args) {
                 .describedAs("name")
                 .ofType(String.class);
             generateNameOpt = parser.accepts("generate-name", "Generate a UUID to use as the name.");
-            intervalOpt = parser.accepts("interval", "The metrics push interval in milliseconds.")
+            String nl = System.lineSeparator();
+
+            intervalOpt = parser.accepts("interval", "The metrics push interval in milliseconds." + nl + "Leave empty to reset the interval.")
                 .withRequiredArg()
                 .describedAs("push interval")
-                .ofType(java.lang.Integer.class);
+                .ofType(String.class);
 
-            String nl = System.lineSeparator();
 
             String[] matchSelectors = new String[] {
                 "client_id", "client_instance_id", "client_software_name",
                 "client_software_version", "client_source_address", "client_source_port"
             };
             String matchSelectorNames = Arrays.stream(matchSelectors).map(config -> "\t" + config).collect(Collectors.joining(nl));
-            matchOpt = parser.accepts("match",  "Matching selector 'k1=v1,k2=v2'. The following is a list of valid selector names: " + nl + matchSelectorNames)
+            matchOpt = parser.accepts("match",  "Matching selector 'k1=v1,k2=v2'. The following is a list of valid selector names: " + nl + matchSelectorNames + nl + "Leave empty to reset the match selector.")
                 .withRequiredArg()
                 .describedAs("k1=v1,k2=v2")
                 .ofType(String.class)
                 .withValuesSeparatedBy(',');
-            metricsOpt = parser.accepts("metrics",  "Telemetry metric name prefixes 'm1,m2'.")
+            metricsOpt = parser.accepts("metrics",  "Telemetry metric name prefixes 'm1,m2'." + nl + "Leave empty to reset the metrics prefixes.")
```

Comment:
I think perhaps this is an unnecessary change. If the list of metric name prefixes is empty, it sets an empty list. This happens to be the default, so it has the effect of "resetting" the prefixes. Personally, I think of it as setting an empty list, which probably doesn't need a change in the usage message.